clicommand/annotate.go

```diff
@@ -94,6 +95,11 @@ var AnnotateCommand = cli.Command{
 			Usage:  "Append to the body of an existing annotation",
 			EnvVar: "BUILDKITE_ANNOTATION_APPEND",
 		},
+		cli.IntFlag{
+			Name:   "priority",
+			Usage:  "Priority of the annotation (1 to 10), default is 3",
```

Comment:
I think you should mention that higher priority annotations will appear above lower ones in the UI. It might be obvious to some, but IMO `--priority 1` is a bit ambiguous.